client/layout/guided-tours/docs/API.md

```diff
@@ -216,3 +216,53 @@ combineTours( {
   thirdTour: ThirdTour,
 } );
 ```
+
+## Targeting elements in Calypso
+
+One of the features of Guided Tours is the ability to target DOM elements in Calypso. Targeting can be achieved in two ways:
+
+- `data-tip-target` attribute on the target element (recommended)
+- using any CSS selector
+
+To find the DOM node, `document.querySelector` is used. By default, the query used is `[data-tip-target="${target}"]`. However, if `target` starts with a dot or a space character, it will be treated like a standard selector and be passed directly to the function, thereby allowing you to target elements that do no set a `data-tip-target` attribute. We run the query selector on every render so it possible to move elements around.
```

Comment:
"... if `target` contains a `.`, `#`, or space character, we assume it's a CSS selector and it will be treated as such ..."

see code: 

```js
if ( targetSlug.indexOf( '.' ) !== -1 || targetSlug.indexOf( '#' ) !== -1 || targetSlug.indexOf( ' ' ) !== -1 ) {
```